Gemfile.global

```diff
@@ -8,7 +8,7 @@ if ENV['TEST_FRAMEWORK'] == 'rspec'
   gem 'rspec', :require => false
 end
 
-gem 'pg', '~> 1.5.3', :platform => [:ruby, :mswin, :mingw]
+gem 'pg', '~> 1.5.3', platforms: [:ruby, :mswin, :mingw]
```

Comment:
Even very old versions of Bundler have this being plural: https://bundler.io/v1.12/man/gemfile.5.html.